src/components/table/table-cell.react.js

```diff
@@ -101,16 +31,27 @@ type TTableCellProps = {
 };
 
 export function TableCell(props: TTableCellProps) {
-  const {active, children, header, order, ...otherProps} = props;
-  const style = pickStyles(otherProps);
+  const {active, children, header, order, className, ...otherProps} = props;
   const icon = `keyboard_arrow_${order === 'desc' ? 'down' : 'up'}`;
-  const Component = header ? StyledTableHeaderCell : StyledTableCell;
+  // const Component = header ? StyledTableHeaderCell : StyledTableCell;
```

Comment:
Q: Is this code commented out to stay?

Reply:
Has been fixed in the flow removal branch 👍